build.sbt

```diff
@@ -275,6 +276,21 @@ lazy val enumeratumArgonaut =
       )
     )
 
+lazy val enumeratumJson4s =
```

Comment:
Can you please add references to this project 

1. [In this list of integration projects](https://github.com/lloydmeta/enumeratum/blob/master/build.sbt#L38)
2. [In this list of 2.12-compatible projects](https://github.com/lloydmeta/enumeratum/blob/master/build.sbt#L79) (I think I saw a 2.12 artefact)

It's a bit of a hassle but we're still waiting on Play to come out with their 2.12 release :)